broker/setup_broker/setup_broker.sh

```diff
@@ -10,7 +10,7 @@ survey="${3:-ztf}"
 # name of the survey this broker instance will ingest
 # 'ztf' or 'decat'
 schema_version="${4:-3.3}"
-versiontag="v$(echo ${schema_version} | tr . _)"  # 3.3 -> v3_3
+versiontag=v$(echo "${schema_version}" | tr . _)  # 3.3 -> v3_3
```

Comment:
And while it's definitely good to quote all these string expansions, is there some point along the way that expects (requires?) that the versiontag not have any whitespace?

Not a question to fix, just a question to consider in your own mind for thinking about shell scripting and documenting behavior for the future.